.cirrus.yml

```diff
@@ -0,0 +1,15 @@
+macos_instance:
+  image: ghcr.io/cirruslabs/macos-monterey-base:latest
+
+task:
+  script: |
+    brew install python@3.10
+    brew install python-tk@3.10
+    /opt/homebrew/opt/python@3.10/bin/python3 -m venv ~/py_310
+    source ~/py_310/bin/activate
+    cd package
+    python -m pip install .
+    cd ../testsuite
+    python -m pip install .
+    python -m pip install pytest
+    python -m pytest MDAnalysisTests
```

Comment:
We may get access to more than 1 core, just not more than 1 "CI job" I think, not sure. May also want to install more deps for testing, but it is a start at least..